src/contracts/Persistence/Content/Location.php

```diff
@@ -73,9 +73,11 @@ class Location extends ValueObject
     public $remoteId;
 
     /**
-     * ID of the corresponding {@link \Ibexa\Contracts\Core\Persistence\Content}.
+     * Content ID.
      *
-     * @var mixed Content ID.
+     * ID of the corresponding {@see \Ibexa\Contracts\Core\Persistence\Content}.
+     *
+     * @var mixed
```

Comment:
Since you touched it... it should be int :D
```suggestion
     * @var int
```